main.py

```diff
@@ -0,0 +1,70 @@
+"""
+You could download multiple courses at once 
+with help of the script from stepic.org .
+
+
+Run by: 
+python3 main.py
+
+Additional notes:
+1) Add courses which you want to download to the "courses" dictionary.
+
+2) You could also set quality of the video by changing "quality" variable. 
+The default quality is 720.
+"""
+
+import os
+
+import weekDownloader
+
+def main():
+    # Set the quality of a video.
+    quality = '720'
+
+    courses = {}
+
+    # Add courses.
+    # courses[course_id] = [week_1, week_2, ...]
+
+    # Course example.
+    courses["154"] = ["1", "2", "3"]
```

Comment:
Now, for running this, you should 
1) pass keys via Environment variables 
2) Edit source code 

I prefer the way when all the data passed through argv or one settings file